sdks/router-sdk/src/entities/mixedRoute/route.ts

```diff
@@ -13,9 +14,10 @@ type TPool = Pair | Pool
  */
 export class MixedRouteSDK<TInput extends Currency, TOutput extends Currency> {
   public readonly pools: TPool[]
-  public readonly path: Token[]
+  public readonly path: Currency[]
   public readonly input: TInput
```

Comment:
we probably want to comment `input` field with @deprecated JSDoc. 

Ideally if we can change input from public to private, that'd be great, but it might break things. Or alternatively, we can remain `input` as public, but rename it to something like `input_deprecated` for awareness.

Reply:
the input field is still used